jobs.html

```diff
@@ -2,18 +2,28 @@
 layout: default
 title: Jobs with Compiler
 ---
-<h1>Jobs</h1>
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-md-8 offset-md-2 mt-5">
+            <span class="label rounded p-2">Careers</span>
+            <h1>Compiler is a woman-owned software consultancy that’s passionate about making government tech solutions
+                accessible for all.</h1>
 
-<h2>Accepting applications</h2>
-<ul id="open" class="mx-3"></ul>
+            <h2 class="mt-5">Open roles</h2>
+            <ul id="open" class="no-bullets text-decoration-none"></ul>
 
-<h2>Past opportunities</h2>
-<ul id="closed" class="mx-3"></ul>
+            <h2 class="mt-5">Past roles</h2>
+            <ul id="closed" class="no-bullets"></ul>
```

Comment:
You can use Bootstrap's `list-unstyled` here instead of `no-bullets`: https://getbootstrap.com/docs/5.2/content/typography/#unstyled

Reply:
Done in [122bbe2](https://github.com/compilerla/compiler.la/pull/87/commits/122bbe2a6ba7a6c002daaac5db6280507a75b2b5)